### DISCUSSION TITLE
LayoutAnimation

### DISCUSSION THREAD

**Discussion by dev-adsoftware** [reactwg/react-native-new-architecture#113] [Q&A] [answered]
I have a project with the new architecture. Everything is working great except LayoutAnimation. I haven't seen anything in any documentation that says if this should or should not be working with the new architecture. As a stripped down example, I tried the LayoutAnimation examples in the official documentation. Running on an iPhone (either in simulator or on the device), the LayoutAnimation is not animating. Is this a known issue?

Without new architecture, same code, works fine.

**Comment by cortinico** [comment, marked as answer]
Thanks for raising this point @dev-adsoftware 
You're right in saying that LayoutAnimations are disabled. We've enabled them for the New Architecture on iOS just recently (https://github.com/facebook/react-native/commit/0a30aa361224639dbec0bbf33351673b67d31e75).

They'll be enabled by default in RN 0.72. In the meanwhile you can udpate the `ReactNativeConfig.cpp` file to turn them on in your project if you wish.

@sammy-SC can also provide more context on this change.
